sql/core/src/main/scala/org/apache/spark/sql/functions.scala

```diff
@@ -1299,6 +1299,70 @@ object functions {
    */
   def toRadians(columnName: String): Column = toRadians(Column(columnName))
 
+  /**
+   * Returns the first argument-base logarithm of the second argument.
+   *
+   * @group math_funcs
+   * @since 1.4.0
+   */
+  def logarithm(l: Column, r: Column): Column = Logarithm(l.expr, r.expr)
```

Comment:
btw I think it is OK to just name this log, and we should move them to where log is right now.